_versions/2021-06-07-opensearch-1.0.0-rc1.markdown

```diff
@@ -13,7 +13,7 @@ components:
   -
     role: ui
     artifact: opensearch-dashboards
-    version: 1.0.0-rc1
+    version: 1.1.0-rc1
```

Comment:
I'm not sure why you're changing this one - this line is causing an error (and it's opensearch dashboard not CLI)

Reply:
Ack.